docs/source/providers/index.md

```diff
@@ -0,0 +1,59 @@
+# Providers Overview
+
+The goal of Llama Stack is to build an ecosystem where users can easily swap out different implementations for the same API. Examples for these include:
+- LLM inference providers (e.g., Fireworks, Together, AWS Bedrock, Groq, Cerebras, SambaNova, etc.),
+- Vector databases (e.g., ChromaDB, Weaviate, Qdrant, FAISS, PGVector, etc.),
+- Safety providers (e.g., Meta's Llama Guard, AWS Bedrock Guardrails, etc.)
+
+Providers come in two flavors:
+- **Remote**: the provider runs as a separate service external to the Llama Stack codebase. Llama Stack contains a small amount of adapter code.
+- **Inline**: the provider is fully specified and implemented within the Llama Stack codebase. It may be a simple wrapper around an existing library, or a full fledged implementation within Llama Stack.
+
+Importantly, Llama Stack always strives to provide at least one fully "local" provider for each API so you can iterate on a fully featured environment locally.
```

Comment:
Do you mean inline instead of local here?

Reply:
Actually this section is taken directly from the existing Core Concepts section.

![Screenshot 2025-02-21 at 4 18 52 PM](https://github.com/user-attachments/assets/b226f06c-bed3-4aae-bf05-8a4c66e3e25d)

Reply:
I imagine it's to give clarity to the unfamiliar about what "inline" means, though it could be more explicit.

Reply:
I think we re-branded "local provider" to "inline provider" so this might be an old reference that we unintentionally left behind. "Local" is also ambiguous since we can set up local servers that connect to remote provider.

Reply:
I can change it in my PR 👍

Reply:
Updated, also updated the Concepts page.